crates/aptos-genesis/src/lib.rs

```diff
@@ -39,6 +40,7 @@ pub struct GenesisInfo {
     pub allow_new_validators: bool,
```

Comment:
should these field be grouped into GenesisConfiguration as well to keep consistency?

Reply:
I'm not quite sure why we have so many different GenesisConfiguration types everywhere either - most likely to  avoid weird dependencies between packages in crate and aptos-move/... I can look into doing a refactor in a follow up PR later